### DISCUSSION TITLE
Album Artwork Missing File Extension in Converted Files

### DISCUSSION THREAD

**Discussion by Virtual-Dynamics** [enzo1982/freac#461] [Help & Support]
I've been using fre:ac for a while and it had been working great until recently.  For some reason it has started dropping the file extension from the embedded album artwork.  If I try to extract the artwork from a converted file with MP3Tag, I get a generic "folder" file instead of "folder.jpg" or "folder.png".  Is there a way to fix this?

Edit: Actually, it looks like there was a bug reported last week for this issue and it has been fixed in the continuous builds.  I must have missed the pop up that fre:ac updated to to the new version.

**Comment by enzo1982** [comment]
Yes, this is related to issue #454 and is fixed already in the continuous builds.

There will be a 1.1.8 update in the near future, but currently I'm still working on other fixes that should go in there.
